test/gtest/ucp/test_ucp_ep_reconfigure.cc

```diff
@@ -92,33 +92,36 @@ class test_ucp_reconfigure : public ucp_test {
 
     bool is_exclude_iface() const
     {
-        return get_variant_value();
+        return get_variant_value(0);
     }
 
     void send_recv()
     {
-        static constexpr unsigned num_iterations = 100;
-        static constexpr size_t msg_size         = 16 * UCS_KBYTE;
-        const ucp_request_param_t param          = {
+        static const unsigned num_iterations = 100;
+        static const size_t msg_size         = 16 * UCS_KBYTE;
+        static const uint64_t seed           = 0xdeadbeef;
+        const ucp_request_param_t param      = {
             .op_attr_mask = UCP_OP_ATTR_FLAG_NO_IMM_CMPL
         };
 
         for (unsigned i = 0; i < num_iterations; ++i) {
-            std::string sbuf(msg_size, 'a'), rbuf(msg_size, 'b');
+            mem_buffer sbuf(msg_size, UCS_MEMORY_TYPE_HOST, seed);

```

Comment:
maybe `seed + 1` to add variability to test

Reply:
you mean seed +i?